### PR TITLE
add: link to replit run

### PR DESCRIPTION
**Pull request by eankeen**
i think it would be pretty cool to let people see how this code works (and edit / preview the program) right in their browser, without any manual downloads or installation. i'm thinking we can add a 'run on repl.it' button, which redirects to something like [this](https://repl.it/@eankeen/run-tape):

image

### CODE REVIEW

**Comment by ljharb** [comment]
We'd not want to link to something that uses an unofficial package installer, so this is a great idea if you can use npm instead of yarn (ie, if you can make the repl.it actually match this repo, which uses npm)

**Comment by eankeen** [comment]
oh gotchia - yes i can match the commands

File: .replit [hunk 1]
```diff
@@ -0,0 +1,2 @@
+language = "nodejs"
+run = "npm install && cd example && node array.js"
```
Comment (ljharb) [inline .replit]:
```suggestion
run = "npm install && cd example && node array"
```
altho i wonder if we should make an `index.js` that requires all the other files, so we can run all the examples here?

**Comment by ljharb** [comment]
@eankeen did you delete your fork? :-( if so, could you try recreating it, so I can land this?

**Comment by ljharb** [comment]
@eankeen I'd still like to land this, but it appears your fork has been deleted.

**Pull request closed without merging**